### PR TITLE
chore(pyroscope): use modules from a merged repository

### PR DESCRIPTION
**Pull request by korniltsev**
#### PR Description
Use Pyroscope modules from https://github.com/grafana/pyroscope 
Old Phlare repo is archived https://github.com/grafana/phlare 

In ebpf there are no changes, only imports.
In api there no functional changes, only imports and proto files regenerated
#### PR Checklist

- [ ] CHANGELOG updated
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -716,4 +718,4 @@ exclude (
 
 replace github.com/github/smimesign => github.com/grafana/smimesign v0.2.1-0.20220408144937-2a5adf3481d3
 
-// replace github.com/grafana/phlare/ebpf => ../phlare/ebpf
+// replace github.com/grafana/pyroscope/ebpf => ../pyroscope/ebpf
```
Comment (ptodev) [inline go.mod:721]: Do we need this comment?

File: go.mod [hunk 2]
```diff
@@ -619,7 +617,11 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
 
-require github.com/pyroscope-io/godeltaprof v0.1.1
+require (
```
Comment (ptodev) [inline go.mod:620]: Do you mind placing this in the first `require` block in the go.mod please? (the "direct" one)

**Review by ptodev — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (korniltsev) [reply go.mod]: nope, it is just useful for development without publishing locally. will remove it to not confuse.

File: go.mod (hunk 2 above)
Comment (korniltsev) [reply go.mod:623]: sure

**Pull request merged**